qa/src/test/java/io/camunda/migrator/qa/element/SubprocessMigrationTest.java

```diff
@@ -25,22 +28,28 @@ public void migrateCallActivityAndSubprocess() {
     // given
     deployProcessInC7AndC8("calledActivitySubprocess.bpmn");
     deployProcessInC7AndC8("callActivityProcess.bpmn");
-    ProcessInstance instance = runtimeService.startProcessInstanceByKey("callingProcessId");
-    ProcessInstance calledInstance = runtimeService
+    ProcessInstance parentInstance = runtimeService.startProcessInstanceByKey("callingProcessId");
+    ProcessInstance subProcessInstance = runtimeService
         .createProcessInstanceQuery()
-        .superProcessInstanceId(instance.getProcessInstanceId())
+        .superProcessInstanceId(parentInstance.getProcessInstanceId())
         .singleResult();
     // when
     runtimeMigrator.start();
 
     // then
-//    TODO https://github.com/camunda/camunda-bpm-platform/issues/5167
-//    assertThat(byProcessId("callingProcessId")).isActive()
-//        .hasActiveElements(byId("callActivityId"))
-//        .hasVariable(LEGACY_ID_VAR_NAME, instance.getProcessInstanceId());
+    List<io.camunda.client.api.search.response.ProcessInstance> processInstances = camundaClient.newProcessInstanceSearchRequest().send().join().items();
+    Optional<io.camunda.client.api.search.response.ProcessInstance> c8ParentInstance = processInstances.stream()
+        .filter(pi -> pi.getProcessDefinitionId().equals("callingProcessId")).findFirst();
```

Comment:
Nit
Instead of getting all instances and then filtering, could just specify a `ProcessInstanceFilter` in the searchrequest to only look for the instance with the given definitionId. 
Since in this test its not gonna be that many instances either way the impact is probably negligible though just looks a bit neater 🤷‍♀️